14/umbraco-cms/fundamentals/backoffice/documenting-your-controllers.md

```diff
@@ -0,0 +1,68 @@
+﻿# Documenting your controllers
+Documenting your API controllers using Swagger in Umbraco Version 14 simplifies the creation of detailed and interactive API documentation. By adding Swagger attributes, you automatically generate comprehensive information about routes, parameters, and response types, enhancing the developer experience and ensuring clarity and consistency in your API documentation.
+
+
+## ApiExplorerSettings
+With the `ApiExplorerSettings` attribute, we can put all of our endpoints into a given group, this is a nice way of organizing our endpoints in the Swagger UI.
+
+```csharp
```

Comment:
Seems like an example is missing here? 🤔

Reply:
Fixed 🚀